OpenRA.Mods.Common/UtilityCommands/ExtractLuaDocsCommand.cs

```diff
@@ -70,17 +70,20 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 				var name = t.GetCustomAttributes<ScriptGlobalAttribute>(true).First().Name;
 				var members = ScriptMemberWrapper.WrappableMembers(t);
 
+				Console.WriteLine();
 				Console.WriteLine("### " + name);
-				Console.WriteLine("<table>");
+				Console.WriteLine();
+				Console.WriteLine("| Function | Description |");
+				Console.WriteLine("|----------|-------------|");
 				foreach (var m in members.OrderBy(m => m.Name))
 				{
 					var desc = m.HasAttribute<DescAttribute>() ? m.GetCustomAttributes<DescAttribute>(true).First().Lines.JoinWith("\n") : "";
```

Comment:
This also needs to change
```suggestion
					var desc = m.HasAttribute<DescAttribute>() ? m.GetCustomAttributes<DescAttribute>(true).First().Lines.JoinWith("<br />") : "";
```
There are two other instances of that in this file, after fixing those as well it renders correctly for me.

Reply:
Replaced all instances.